README.md

```diff
@@ -77,36 +83,53 @@ npm install
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-## :busts_in_silhouette: Authors <a name="authors"></a>
-:bust_in_silhouette: **Harriet Oteng**
+## 👥 Authors <a name="authors"></a>
+
+👤 **Harriet Oteng**
+
 - GitHub: [@githubhandle](https://github.com/newhoteng)
 - Twitter: [@twitterhandle](https://twitter.com/HarrietOteng1)
-- LinkedIn: [LinkedIn](https://www.linkedin.com/in/harriet-oteng-75554666/)
+- LinkedIn: [Harriet Oteng](https://www.linkedin.com/in/harriet-oteng-75554666/)
+
+👤 **Oumaima Nejjari**
+
+- GitHub: [@aima98](https://github.com/aima98)
+- LinkedIn: [Oumaima Nejjari](https://www.linkedin.com/oumaimanejjari/)
+
+👤 **Aimable Bacunguye**
+
+- GitHub: [@githubhandle](https://github.com/BANCUNGUYE66)
+- LinkedIn: [Aimable Bacunguye](https://www.linkedin.com/in/aimable-bancunguye-aba703143/)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 ## :telescope: Future Features <a name="future-features"></a>
-  - Category Page
+
```

Comment:
Please include Adding a mobile version.

Reply:
Thank you for taking the time to review my code. I added what you mentioned to our README file.